src/methods/defining_methods/hard/q9/NumCoursesMethod.java

```diff
@@ -0,0 +1,26 @@
+package methods.defining_methods.hard.q9;
+
+/**
+ * [J] Write a Java Program that asks users for the number of courses they are taking right now. A course will contain
+ * 3 credits. Write a method named creditsCac that accepts the number of courses and returns the number of credits the student will receive.
+ * 
```

Comment:
We should also probably have a return -1 if the number of credits is less than 0